com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickPool.cs

```diff
@@ -104,12 +162,15 @@ internal ProbeBrickPool(ProbeVolumeTextureMemoryBudget memoryBudget, ProbeVolume
 
             m_FreeList = new Stack<BrickChunkAlloc>(256);
 
-            int width, height, depth;
-            DerivePoolSizeFromBudget(memoryBudget, out width, out height, out depth);
-            int estimatedCost = 0;
-            m_Pool = CreateDataLocation(width * height * depth, false, shBands, "APV", out estimatedCost);
+            DerivePoolSizeFromBudget(memoryBudget, out int width, out int height, out int depth);
+            m_Pool = CreateDataLocation(width * height * depth, false, shBands, "APV", true, out int estimatedCost);
             estimatedVMemCost = estimatedCost;
 
+            // Allocate two pools for state blending. They contain states data and interpolated result is written in m_Pool
+            // TODO: they shouldn't allocate a validity texture because no blending is needed on validity
```

Comment:
linked to https://github.com/Unity-Technologies/Graphics/pull/7020#discussion_r800588603